postprocessor/src/main/java/com/azure/autorest/postprocessor/Postprocessor.java

```diff
@@ -237,4 +248,147 @@ private String getBaseDirectory() {
             Utils.deleteDirectory(tempDirWithPrefix.toFile());
         }
     }
+
+    private void handlePartialUpdate(Map<String, String> fileContents) {
```

Comment:
Why not put them to a separate class? At least it will be easier for your unit test later.

Reply:
Agree, I also plan to do this refactoring, just first check in to show the logic.

Reply:
Done  with refactoring